scipy/linalg/flapack.pyf.src

```diff
@@ -1243,7 +1397,7 @@ interface
 
    end subroutine <prefix2c>gesvd
 
-   subroutine <prefix2c>gesvd_lwork(m,n,minmn,u0,u1,vt0,vt1,a,compute_uv,full_matrices,u,s,vt,work,rwork,lwork,info)
```

Comment:
This may be correct, but is above my ``f2py`` fu. Why can this be removed here?

Reply:
In LAPACK lwork routines, the routine performs an early exit when `lwork=-1` and only needs a few mandatory inputs such `m`, `n` here. Then the optimal work array length is saved as a number in the first element of `work`. 

But because these variables are not used in the original call statement the compiler during the build makes some noise about these variables are not used. Hence, I've removed them. There are still some noise left about `lange` routines during the build but that seemed too delicate for a simple surgery.

Reply:
OK thanks for the explanation. Always good to get rid of build noise:)